jobs/build/rhcos_sync/Jenkinsfile

```diff
@@ -46,18 +46,18 @@ node {
                     commonlib.ocpVersionParam('OCP_VERSION', '4', ['auto']),
                     string(
                         name: 'OVERRIDE_BUILD',
-                        description: 'ID of the RHCOS build to sync. e.g.: <b>42.80.20190828.2</b>. This overrides what would be inferred from <code>FROM_RELEASE_TAG</code> and requires explicit specifications below too.',
-                        defaultValue: "",
+                        description: "ID of the RHCOS build to sync. e.g.: <code>42.80.20190828.2</code>. Leave at <code>auto</code> to have the metadata extracted from the installer image. <b>Leave at <code>auto</code> unless you know what you're doing</b>",
+                        defaultValue: "auto",
                         trim: true,
                     ),
                     choice(
                         name: 'ARCH',
-                        description: 'Which architecture of RHCOS build to look for. Required with <code>OVERRIDE_BUILD</code>',
+                        description: 'Which architecture of RHCOS build to look for. Required with <code>OVERRIDE_BUILD</code>. Leave at <code>auto</code> otherwise.'
                         choices: (["auto"] + commonlib.brewArches),
                     ),
                     string(
                         name: 'OVERRIDE_NAME',
-                        description: 'The release name, like <b>4.2.0</b>, or <b>4.2.0-0.nightly-2019-08-28-152644</b>. Required with <code>OVERRIDE_BUILD</code>',
+                        description: 'The release name, like <b>4.2.0</b>, or <b>4.2.0-0.nightly-2019-08-28-152644</b>. Required with <code>OVERRIDE_BUILD</code>. Leave at <code>auto</code> otherwise.',
```

Comment:
This could be confusing, override_build instruction should be good enough to scare someone and make them double check